src/deserialization.hpp

```diff
@@ -123,7 +123,7 @@ Status deserializePredictRequest(
 
             if (blob == nullptr) {
                 Status status = StatusCode::OV_UNSUPPORTED_DESERIALIZATION_PRECISION;
-                SPDLOG_ERROR(status.string());
+                SPDLOG_DEBUG(status.string());
```

Comment:
in this case it makes sense to use spdlog::debug (we loose the line & file but we still can debug OVMS with different CLI option without need to rebuild (request that from client).